excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/validation/CategoryValidatorTest.java

```diff
@@ -0,0 +1,252 @@
+    private static final String CATEGORY_LABEL = "evidence";
+    private static final Integer DISPLAY_ORDER = 100;
+    protected static final String NO_PARENT = "";
+    private static final String DUPLICATED_CATEGORY_ID_ERROR = "cannot be duplicated within case type.";
+    private static final String DISPLAY_ERROR = "DisplayOrder cannot be duplicated";
+    private static final String CASE_TYPE2 = "FT_MultiplePages";
+    private static final String ERROR_CASE_TYPE_PARENT = "CategoryTab Invalid ParentCategoryID";
+    private static final String CASE_TYPE_NOT_FOUND = "It cannot be found in the spreadsheet";
+    private static final String INVALID_VALUE = "CategoryTab Invalid value";
+
+    private CategoryValidator categoryValidator;
+    private DefinitionDataItem definitionDataItem;
+
+    @Before
+    public void setup() {
+        MockitoAnnotations.initMocks(this);
+        parseContext = buildParseContext();
+        categoryValidator = new CategoryValidator();
+        definitionDataItem = buildDefinitionDataItem(LIVE_FROM, LIVE_TO, CASE_TYPE, CATEGORY_ID,
+            CATEGORY_LABEL, DISPLAY_ORDER, NO_PARENT);
+    }
+
+    protected ParseContext buildParseContext() {
+        val parseContext = new ParseContext();
+        val caseTypeEntity1 = new CaseTypeEntity();
+        val caseTypeEntity = new CaseTypeEntity();
+        caseTypeEntity.setReference(CASE_TYPE);
+        caseTypeEntity1.setReference(CASE_TYPE2);
+        parseContext.registerCaseType(caseTypeEntity);
+        parseContext.registerCaseType(caseTypeEntity1);
+        return parseContext;
+    }
+
+    protected DefinitionDataItem buildDefinitionDataItem(String liveFrom, String liveTo, String caseType,
+                                                         String categoryID, String categoryLabel, Integer displayOder,
+                                                         String parentCategoryId) {
+
+        val definitionDataItem = new DefinitionDataItem(SheetName.CATEGORY.toString());
+        definitionDataItem.addAttribute(ColumnName.LIVE_FROM, liveFrom);
+        definitionDataItem.addAttribute(ColumnName.LIVE_TO, liveTo);
+        definitionDataItem.addAttribute(ColumnName.CASE_TYPE_ID, caseType);
+        definitionDataItem.addAttribute(ColumnName.CATEGORY_ID, categoryID);
+        definitionDataItem.addAttribute(ColumnName.DISPLAY_ORDER, displayOder);
+        definitionDataItem.addAttribute(ColumnName.CATEGORY_LABEL, categoryLabel);
+        definitionDataItem.addAttribute(ColumnName.PARENTCATEGORY_ID, parentCategoryId);
+        return definitionDataItem;
+    }
+
+    // Example 1- validly defined Categories tab is as below
+    @Test
+    public void testValidateCategoryTab() {
+        categoryValidator.validate(parseContext, Lists.newArrayList(definitionDataItem));
+    }
+
+    // Example 6 - Invalid definition - Same categoryID defined in the sub-categories
+    @Test(expected = InvalidImportException.class)
```

Comment:
For all the tests that check an exception this way - could use `assertThrows()` in the test instead, a bit cleaner. Or at least check the exception is an `InvalidImportException` in the `catch` that is already there, instead of re-throwing.